backend/src/controllers/reports/helpers/index.js

```diff
@@ -152,19 +152,34 @@ const getHeaderInfoFrom = (templateType = "docx") => {
 const getReport = getReportAndSetRequestHeaders();
 
 /**
- * Groups an array of objects by a specified property.
+ * Groups items in a list by a specified property.
  *
- * @param   {object[]} rows - The array of objects to group.
- * @param   {string}   prop - The property to group by.
- * @returns {object[]}      Returns an array of objects grouped by the specified property.
+ * @param   {Array}  list  - The list of items to be grouped.
+ * @param   {string} group - The property name to group the items by. Default is "portfolio_name".
+ * @returns {Array}        An array of objects, where each object represents a group and its associated items.
+ *                         Each object has two properties: the specified 'group' property (key) and the array of items (value).
  */
-const groupByProperty = (rows, prop) =>
-  _.isEmpty(rows)
-    ? rows
-    : _.map(_.groupBy(rows, prop), (value, key) => ({
-        [prop]: key,
-        projects: [...value],
-      }));
+const groupByProperty = (list = [], group = "portfolio_name") => {
+  // Using lodash's chain function "_" to create a chainable sequence.
+  // The input 'list' is wrapped in the chain and will be operated upon.
+  const result = _(list)
```

Comment:
With lodash commented-out above, will this work?

Reply:
I took a look and found a working import at the top of this file